abel/tools/circularize.py

```diff
@@ -0,0 +1,291 @@
+        pre-center image using :func:`abel.tools.center.center_image`
+        "com", "convolution", "gaussian", "image_center", "slice", or
+        float tuple center (y, x).
+
+    radial_range : tuple, or None
+        limit slice comparison to the radial range tuple (rmin, rmax), in
+        pixels, from the image center. Use to determine the distortion 
+        correction associated with particular peaks. 
+
+    dr : float
+        Radial grid size for the polar coordinate image, default = 0.5 pixel.
+        This is passed to :func:`abel.tools.polar.reproject_image_into_polar`.
+
+        Small values may improve the distortion correction, which is often of
+        sub-pixel dimensions, at the cost of reduced signal to noise for the
+        slice intensity profile. 
+
+    dt : float
+        Angular grid size. This sets the number of radial slices, given by
+        `2*np.pi/dt`. Default = 0.1, ~ 63 slices. More slices, smaller `dt`,
+         may provide a more detailed angular variation of the correction,
+         at the cost of greater signal to noise in the correction function.
+
+        Also passed to :func:`abel.tools.polar.reproject_image_into_polar`
+
+    smooth : float
+        This value is passed to the `scipy.interpolate.UnivariateSpline`
+        function and controls how smooth the spline interpolation is. A value 
+        of zero corresponds to a spline that runs through all of the points,
+        and higher values correspond to a smoother spline function. 
+
+        It is important to examine the relative peak position (scaling factor)
+        data and how well it is represented by the spline function. Use the
+        option `return_correction=True` to examine this data. Typically,
+        `smooth` may remain zero, noisy data may require some smoothing.
+
+    ref_angle : `None` or float
+        Reference angle for which radial coordinate is unchanged.
+        Angle varies between -pi to pi, with zero angle vertical.
+
+        `None` uses numpy.mean(radial scale factors), which attempts to maintain
+        the same average radial scaling. This approximation is likely valid,
+        unless you know for certain that a specific angle of your image
+        corresponds to an undistorted image.
+
+    inverse : bool
+        Inverse Abel transform the *polar* image, to remove the background
+        intensity. This may improve the signal to noise, allowing weak 
+        intensity featured to be followed in angle. 
+
+        Note that this step is only for the purposes of allowing the algorithm
+        to better follow peaks in the image. It does not affect the final 
+        image that is returned, expect for (hopefully) slightly improving the
+        precision of the distortion correction.
+
+    return_correction : bool
+        Additional outputs, as describe below.
+
+    Returns
+    -------
+    IMcirc : numpy 2D array, same size as input
+        Circularized imput image
```

Comment:
"Circularized version of the input image"
(or, at least change "imput" to "input")